apps/omg_watcher/test/omg_watcher/integration/invalid_exit_test.exs

```diff
@@ -65,9 +65,34 @@ defmodule OMG.Watcher.Integration.InvalidExitTest do
     IntegrationTest.wait_for_byzantine_events([%Event.InvalidExit{}.name, %Event.InvalidExit{}.name], @timeout)
 
     # after the notification has been received, a challenged is composed and sent
-    _challenge = TestHelper.get_exit_challenge(first_tx_blknum, 0, 0)
-
-    # TODO: git blame here to bring back removed failing test code, when challenges are re-integrated with the contract
+    challenge = TestHelper.get_exit_challenge(first_tx_blknum, 0, 0)
+
+    assert {:ok, %{"status" => "0x1"}} =
+             OMG.Eth.RootChainHelper.challenge_exit(
```

Comment:
single pipe

Reply:
same here, not in this PR. This test is the same as the test removed in the previous commit, which is the point I'm trying to make here. If it wasn't removed in the prior commit, we'd see this in the diff here (you can still see this, if you search for the right diff - against pre-ALD `master`).